.jenkins/test.sh

```diff
@@ -91,7 +94,7 @@ for test in ./test/*; do
 done
 
 # Get the relative path to where the caffe2 python module was installed
-CAFFE2_PYPATH="$SITE_DIR/caffe2"
+CAFFE2_PYPATH="$INSTALL_SITE_DIR/caffe2"
```

Comment:
These changes needed to work with conda's different prefix in the site-packages name